src/azure-cli/azure/cli/command_modules/acr/commands.py

```diff
@@ -294,3 +302,11 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
 
     with self.command_group('acr token credential', acr_token_credential_generate_util) as g:
         g.command('generate', 'acr_token_credential_generate')
+
+    with self.command_group('acr agentpool', acr_agentpool_util, is_preview=True) as g:
+        g.command('create', 'acr_agentpool_create')
+        g.command('update', 'acr_agentpool_update')
+        g.command('delete', 'acr_agentpool_delete')
+        g.command('list', 'acr_agentpool_list')
+        g.command('show', 'acr_agentpool_show')
```

Comment:
How about g.show_command()?

Reply:
done. Do you know what is the difference? they both seem to work when I tested.

Reply:
https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#register-commands